megamek/src/megamek/client/ui/dialogs/AbstractIconChooserDialog.java

```diff
@@ -105,21 +116,18 @@ protected Container createCenterPane() {
 
     @Override
     protected JPanel createButtonPanel() {
-        JPanel panel = new JPanel(new FlowLayout(FlowLayout.CENTER, 20, 2));
-        panel.setBorder(new EmptyBorder(5, 5, 5, 5));
+        final JPanel panel = new JPanel(new GridLayout(1, 3));
+        panel.setName("buttonPanel");
```

Comment:
Gridlayout is likely to stretch the buttons. I notice you seem to like that or not care; Modern UIs including IntelliJ itself or this gituhub don't seem to do that. On small windows, the buttons float on the bottom right, on bigger ones bottom left or center.

Reply:
I think it works well for our style of application, and that's why I used it in the base components and swapped it over here to follow their formatting.